### PR TITLE
[11.0] [ADD] product_price_history

### PR DESCRIPTION
**Pull request by carlos-domatix**
This module allows you to record cost and sale prices of products.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks for this new module, Carlos. AFAIK, cost price is already stored in a historized table.

**Comment by carlos-domatix** [comment]
Yes. In this module, only the view is created to show this data.

**Comment by pedrobaeza** [comment]
OK, then you should be more specific in the README. You should also follow full README template or use the new readme by fragments system for only writing relevant pieces and let the script to do the rest: https://github.com/OCA/maintainer-tools#readme-generator

**Comment by carlos-domatix** [comment]
Readme generated.

**Comment by pedrobaeza** [comment]
Thanks!

File: product_price_history/models/product.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                    'lst_price': variant.lst_price,
+                    'company_id': self._context.get(
+                        'force_company', self.env.user.company_id.id),
+                })
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    def view_product_price_history(self):
```
Comment (nikul-serpentcs) [inline product_price_history/models/product.py:67]: @carlosDomatix IMO add `api`

File: product_price_history/models/product.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        return {
+            'name': _('Cost Price History'),
+            'view_type': 'form',
+            'view_mode': 'tree',
+            'res_model': 'product.price.history',
+            'type': 'ir.actions.act_window',
+            'domain': [('product_id', '=', self.id)],
+        }
+
+    def view_product_lst_price_history(self):
```
Comment (nikul-serpentcs) [inline product_price_history/models/product.py:79]: same here

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Comment by rousseldenis** [comment]
@carlos-domatix Is this still applicable as there is a name conflict with https://github.com/OCA/product-attribute/tree/12.0/product_price_history ?

**Comment by rousseldenis** [comment]
Closing this as very old. Feel free to reopen it if needed.

**Pull request closed without merging**